mfes/content/src/components/contentFilter.tsx

```diff
@@ -148,23 +150,23 @@ const FilterDialog = ({
                       selected
                         .map((selectedValue: any) => {
                           const selectedOption = options.find(
-                            (option: any) => option.value === selectedValue
+                            (option: any) => option.label === selectedValue
```

Comment:
Please check if this is correct or not. The expected value is 'value,' so why use 'label'? Do not change project-specific changes.